eng/pipelines/common/sign.yml

```diff
@@ -22,4 +24,28 @@ stages:
             **/signed/*.nupkg
           artifactPath: signed
           propsArtifactName: nuget
-          signType: Real
\ No newline at end of file
+          signType: Real
+          preConvertSteps:
+          - task: NuGetCommand@2
+            displayName: Download Microsoft.Maui.Graphics
+            inputs:
+              command: custom
+              arguments: >-
+                install Microsoft.Maui.Graphics -Prerelease -Version $(MauiGraphicsVersion)
+                -OutputDirectory $(Build.StagingDirectory)\maui-graphics
+                -Source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json
+          - task: NuGetCommand@2
+            displayName: Download Microsoft.Maui.Graphics.Win2D.WinUI.Desktop
+            inputs:
+              command: custom
+              arguments: >-
+                install Microsoft.Maui.Graphics.Win2D.WinUI.Desktop -Prerelease -Version $(MauiGraphicsVersion)
+                -OutputDirectory $(Build.StagingDirectory)\maui-graphics
+                -Source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json
+                -Source https://api.nuget.org/v3/index.json
```

Comment:
Do we need both sources here?

Reply:
The NuGet.org source was needed to install some of the Microsoft.Maui.Graphics.Win2D.WinUI.Desktop dependencies that were not on the dotnet6 feed.